mesheryctl/internal/cli/root/relationships/relationship.go

```diff
@@ -31,11 +31,18 @@ import (
 )
 
 var (
-	outFormatFlag string
-
-	availableSubcommands = []*cobra.Command{ViewRelationshipsCmd, GenerateRelationshipDocsCmd}
+	outFormatFlag        string
+	maxRowsPerPage       = 25
+	availableSubcommands = []*cobra.Command{ViewRelationshipsCmd, GenerateRelationshipDocsCmd, SearchComponentsCmd}
 )
 
+type MeshmodelRelationshipsAPIResponse struct {
```

Comment:
@Jougan-0  These structs should be already defined in meshkit  ? // @MUzairS15

Reply:
@aabidsofi19 the type is in [meshery/server/models/meshery_meshmodels_api_response.go](https://github.com/meshery/meshery/blob/7818e6fefae8deaa6a09961b6d0a984c0557ee6b/server/models/meshery_meshmodels_api_response.go#L41)
but it has the last type as entity.entity and the unmarshall was causing the issue for me .
If it is there is meshkit I am unaware of it.:smiling_face_with_tear: